2repository/7.1/community-7.1.1/overload.gradle

```diff
@@ -0,0 +1,23 @@
+ext {
+    alfresco = [
+            version: [
+                    major: 7,
+                    minor: 1,
+                    rev: 1
+            ],
+            flavor: 'community'
+    ]
+    solrimage = 'docker.io/xenit/alfresco-solr6:2.0.2'
+    shareimage = 'docker.io/xenit/alfresco-share-community:7.1.0'
+    psqlimage = 'docker.io/xenit/postgres:13.3'
+    tests = true
+}
+
+dependencies {
+    alfrescoAmp 'org.alfresco:alfresco-share-services:12.24@amp'
+
+    baseAlfrescoWar platform('org.alfresco:acs-community-packaging:7.1.1')
+    baseAlfrescoWar 'org.alfresco:content-services-community@war'
+
+    sharedLib(group: 'org.postgresql', name: 'postgresql', version: '42.2.19')
```

Comment:
Consider upgrading the driver? https://jdbc.postgresql.org/

Reply:
I looked at the supported platofrms page: https://docs.alfresco.com/content-services/latest/support/
If you look for "PostgreSQL 13.3" you will see that there is a note that says "Postgresql-42.2.19.jar".

Reply:
should be ok to upgrade to a higher version, but let's do that in a separate PR?